cmd/occollector/app/builder/exporters_builder.go

```diff
@@ -205,9 +207,9 @@ func typeMismatchErr(
 	dataType configmodels.DataType,
 ) error {
 	return fmt.Errorf(
-		"pipeline %q produces %q to exporter %s which does not support %q "+
-			"telemetry data. exporter will be detached from pipeline",
-		requiredByPipeline.Name, dataType.GetDataTypeStr(),
-		config.Name(), dataType.GetDataTypeStr(),
+		"pipeline %s is attached %s to exporter %s which does not support %s "+
```

Comment:
can you give an example of this message? It may benefit from a re-work

Reply:
> receiver examplereceiver does not support traces but some pipelines that want to process traces are attached to the receiever

Can you suggest how to re-word this to be nicer? I also do not quite like it.

Reply:
What about "receiver exmaplereceiver does not support traces but it was used in some traces pipelines"? As for the pipelines one "examplepipeline is a metrics pipeline but has a exampleexporter which does not support metrics"

Reply:
Or perhaps remove the plural on the first one: "receiver exmaplereceiver does not support traces but it was used in a traces pipeline" (it may have been more than one, but at least one).

Reply:
SGTM

Reply:
Fixed.